pennylane/ops/qubit/parametric_ops.py

```diff
@@ -1272,24 +1354,22 @@ def compute_matrix(theta):  # pylint: disable=arguments-differ
         c = qml.math.cos(theta / 2)
         s = qml.math.sin(theta / 2)
 
-        if interface == "torch":
-            # Use convert_like to ensure that the tensor is put on the correct
-            # Torch device
-            z = qml.math.convert_like(qml.math.zeros([4]), theta)
-        else:
-            z = qml.math.zeros([4], like=interface)
```

Comment:
How comet his was deleted?

Reply:
If I'm not mistaken (I only was able to check with the `device="meta"` example) the device is considered in `zeros_like`, which is used below :)